### PR TITLE
Document more battle anim effects

### PR DESCRIPTION
**Pull request by huderlem**

### CODE REVIEW

File: include/battle_anim.h [hunk 1]
```diff
@@ -119,12 +119,12 @@ void SetSpriteRotScale(u8 spriteId, s16 xScale, s16 yScale, u16 rotation);
 void InitSpriteDataForLinearTranslation(struct Sprite *sprite);
 void PrepareBattlerSpriteForRotScale(u8 spriteId, u8 objMode);
 void SetBattlerSpriteYOffsetFromRotation(u8 spriteId);
-u32 sub_80A75AC(u8 a1, u8 a2, u8 a3, u8 a4, u8 a5, u8 a6, u8 a7);
-u32 sub_80A76C4(u8 a1, u8 a2, u8 a3, u8 a4);
+u32 GetBattleBgPalettesMask(u8 a1, u8 a2, u8 a3, u8 a4, u8 a5, u8 a6, u8 a7);
+u32 GetBattleMonSpritePalettesMask(u8 a1, u8 a2, u8 a3, u8 a4);
```
Comment (GriffinRichards) [inline include/battle_anim.h:123]: Could update these arg names too

File: src/battle_anim_utility_funcs.c [hunk 2]
```diff
@@ -29,16 +29,16 @@ static void AnimTask_HardwarePaletteFade_Step(u8);
...
 static void AnimTask_Flash_Step(u8);
-static void sub_81175C4(u32, u16);
+static void SetPalettesToColor(u32, u16);
 static void AnimTask_UpdateSlidingBg(u8);
-static void sub_8117A60(u8);
+static void UpdateMonScrollingBgMask(u8);
 static void AnimTask_WaitAndRestoreVisibility(u8);
 
-const u16 gUnknown_08597418 = RGB(31, 31, 31);
+static const u16 wCurseLinesPalette[] = { RGB_WHITE };
```
Comment (GriffinRichards) [inline src/battle_anim_utility_funcs.c:41]: Prefix `s` for static data

File: src/battle_anim_utility_funcs.c [hunk 3]
```diff
@@ -756,8 +756,6 @@ void AnimTask_GetTargetIsAttackerPartner(u8 taskId)
     DestroyAnimVisualTask(taskId);
 }
 
-#define tInvisible gBattleAnimArgs[0];
```
Comment (GriffinRichards) [inline src/battle_anim_utility_funcs.c:759]: Why undo this?

File: src/battle_anim_psychic.c [hunk 4]
```diff
@@ -836,7 +836,7 @@ static void AnimTask_ImprisonOrbs_Step(u8 taskId)
     }
 }
 
-static void sub_810FB60(struct Sprite *sprite)
+static void AnimRedx_Step(struct Sprite *sprite)
```
Comment (GriffinRichards) [inline src/battle_anim_psychic.c:839]: `AnimRedX_Step`

**Review by GriffinRichards — COMMENTED** [review]
Plus the build fix, obviously

File: src/battle_anim_utility_funcs.c (hunk 2 above)
Comment (huderlem) [reply src/battle_anim_utility_funcs.c:41]: Too much gbc disassembly

**Pull request merged**